src/Form/test-cases/1337x_signup.html

```diff
@@ -0,0 +1,580 @@
+<!-- https://www.1337x.to/register -->
+<form method="post" id="contact-form" novalidate="true">
+    <div class="form-row login-field">
+        <h4>Username:</h4>
+        <input name="username" type="text" value="" class="form-control" data-manual-scoring="username">
+        <small class="note">Username should be 4-12 characters long (only a-z and 0-9 characters). No special characters
+            or spaces.</small>
+        <br>
+        <small class="note">Valid: ( john1337 , JoHn1337 , JOHN1337 )</small>
+        <br>
+        <small class="note">Invalid: (john 1337 , john$1337 , john! ....)</small>
+    </div>
+    <div class="form-row login-field">
+        <h4>Password:</h4>
+        <input name="password" type="password" value="" class="form-control"
+            data-manual-scoring="password">
+    </div>
+    <div class="form-row login-field">
+        <h4>Retype Password:</h4>
+        <input name="repassword" type="password" value="" class="form-control"
+            data-manual-scoring="password">
+    </div>
+    <div class="form-row login-field">
+        <h4>Email:</h4>
+        <input name="email" type="text" value="" class="form-control"
+            data-manual-scoring="emailAddress">
+    </div>
+    <div class="form-row">
+        <h4>Country</h4>
+        <div class="fancy-select"><select name="country" class="select fancified"
```

Comment:
@GioSensation This is the same situation as Steam - a custom select input UI with a hidden input inside. In the previous PR, you changed Steam's input to `unknown`, but given https://app.asana.com/0/0/1201973119426328/f, should this be `addressCountryCode` instead?

Reply:
In this case, this should be `addressCountryCode` because it's an actual `select` element (not `input type=hidden`) and what's more important autofill does indeed update the visual control. One weird thing I noticed, though, is that we are matching "Haiti" for an identity with country set to "Slovenia". Do you mind taking a look to see if you can understand what's going on there?

Reply:
Will fix the input type.

Haiti is:
```
<option value="164">Haiti</option>
```
Slovenia is:
```
<option value="64">Slovenia</option>
```

[setValueForSelect](https://github.com/duckduckgo/duckduckgo-autofill/blob/5a77445c90909f272cd0a8f9d222a5ac9a1b6e09/dist/autofill.js#L8379) gets called with `el, "64"`, and then does:

```
if (option.value.includes(String(val))) {
```
...which is going to make Haiti match.

Reply:
Yeah, good catch. I have updated this in another branch. It should be pretty safe because we try to format that `val` before this, so strict equality should work here.